Form/TweetValueTransformer.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+namespace EzSystems\TweetFieldTypeBundle\Form;
+
+use Symfony\Component\Form\DataTransformerInterface;
+use eZ\Publish\API\Repository\FieldType;
+use eZ\Publish\Core\FieldType\Value;
+
+class TweetValueTransformer implements DataTransformerInterface
+{
+    /**
+     * @var FieldType
+     */
+    private $fieldType;
+
+    public function __construct(FieldType $fieldType)
+    {
+        $this->fieldType = $fieldType;
+    }
+
+    public function transform($value)
```

Comment:
Yeah, that was puzzling me to, but look at `EzSystems\RepositoryForms\FieldType\DataTransformer`. Some of them are using from/to Hash (FieldValue) , some have just logic inside them (almost all of them) and some are mixed just like this (RichText). It won't work with calling toHash() it this case, because array is expected for storage and string is expected for template. Unless something like toHash()['url'] but that just seems little bit ugly.